src/opendune.c

```diff
@@ -1155,9 +1155,14 @@ static bool OpenDune_Init(int screen_magnification, VideoScaleFilter filter, int
 		return false;
 	}
 
+	g_screen_magnification = screen_magnification;
+	g_scale_filter = filter;
+
 	Timer_Init();
 
-	if (!Video_Init(screen_magnification, filter)) return false;
+#if !(defined(_WIN32) && (defined(WITH_SDL) || defined(WITH_SDL2)))
+	if (!Video_Init()) return false;
+#endif
```

Comment:
Since SDL_Init under Windows must be used in same thread of Video_Tick, where inputs are recognized, I added SDL_Init call inside Video_Tick with Win32.